src/cache/cache_las.c

```diff
@@ -808,10 +834,10 @@ __wt_las_cursor_position(WT_SESSION_IMPL *session, WT_CURSOR *cursor, uint32_t b
      */
     for (;;) {
         WT_CLEAR(las_key);
-        cursor->set_key(cursor, btree_id, key, timestamp, (uint64_t)0);
+        cursor->set_key(cursor, btree_id, key, timestamp, WT_TXN_MAX);
```

Comment:
Can we move the `WT_CLEAR` down to where the variable is used?

Reply:
Done.